Makefile

```diff
@@ -39,6 +38,11 @@ ifeq "$(strip $(VERSION))" ""
  override VERSION = $(shell git describe --always --tags --dirty)
 endif
 
+# Force using Go Modules and always read the dependencies from
+# the `vendor` folder.
+export GO111MODULE = on
```

Comment:
This might cause issues if you are compiling another go project in the same terminal.  will defer it to @nkubala if he had it this way for specific reason.

Reply:
No. The variable will be set only for the Makefile.
Also, with linux, a process can't change the env variables of a parent -> The makefile can't change the env variables of the terminal.

Reply:
I *think* it should be fine, but worst case scenario you can always just unset the variable if you're gonna build something else in that terminal. also this would only be an issue for go versions 1.11 and older.